### PR TITLE
Added --replace flag for convox env set

### PR DESCRIPTION
**Pull request by radekl**

### CODE REVIEW

File: pkg/cli/env.go [hunk 1]
```diff
@@ -188,10 +196,7 @@ func EnvSet(rack sdk.Interface, c *stdcli.Context) error {
 		c.Writer().Stdout = c.Writer().Stderr
 	}
 
-	env, err := helpers.AppEnvironment(rack, app(c))
-	if err != nil {
-		return err
-	}
+	env, err := loadEnvs(rack, c, c.Bool("replace"))
```
Comment (ddollar) [inline pkg/cli/env.go]: This is no longer checking the error return value.

I think you don't need the extra function here, maybe something like:

```golang
env := structs.Environment{}
var err error

if !c.Bool("replace") {
  env, err = helpers.AppEnvironment(rack, app(c))
  if err != nil {
    return err
  }
}

File: pkg/cli/env.go [hunk 2]
```diff
@@ -46,6 +46,7 @@ func init() {
 			flagId,
 			flagRack,
 			flagWait,
+			flagReplace,
```
Comment (ddollar) [inline pkg/cli/env.go]: Sorry one more thing. The `flag*` globals are really only meant for variables that are used in more than one command. Could you please change this to look more like the promote flag below (and alphabetize below it)

**Comment by radekl** [comment]
Fixed both tests. Strange thing that before tests were fine.

**Comment by ddollar** [comment]
Rebased as #3085

**Pull request closed without merging**